src/pint/scripts/pintempo.py

```diff
@@ -43,7 +43,13 @@ def main(argv=None):
     log.warning(m.params)
 
     log.info("Reading TOAs")
-    t = pint.toa.get_TOAs(args.timfile)
+    use_planets = False
+    if m.PLANET_SHAPIRO.value:
+        use_planets = True
+    model_ephem = "DE421"
+    if m.EPHEM is not None:
+        model_ephem = m.EPHEM.value
```

Comment:
Why are these defaults set up in pintempo? Are they different from the defaults get_TOAs would assume? Wait, why does loading TOAs depend on the par file in use?

Reply:
Because TOA objects compute the posvels at each TOA time. This requires knowing what the ephemeris is and whether the planet positions are needed. These are specified in the par file, as a consequence of the unfortunate mixing of model parameters and 'control' parameters in TEMPO par files.

Reply:
We should probably discuss what can be done to clean up this mess. For example, we might have the par file reader return two objects, one for the "control" parameters and one for the actual model parameters. But obviously it's not something you can fix in this PR.

Reply:
Yes, this has been the subject of lots of previous discussion, e.g. https://github.com/nanograv/PINT/wiki/Discussion-of-Control-Parameters

But we haven't come up with a good solution.